hydroflow_datalog_core/src/grammar.rs

```diff
@@ -9,11 +9,25 @@ pub mod datalog {
 
     #[derive(Debug, Clone)]
     pub enum Declaration {
-        Input(#[rust_sitter::leaf(text = ".input")] (), Ident),
-        Output(#[rust_sitter::leaf(text = ".output")] (), Ident),
+        Input(#[rust_sitter::leaf(text = ".input")] (), Ident, RustSnippet),
+        Output(
+            #[rust_sitter::leaf(text = ".output")] (),
+            Ident,
+            RustSnippet,
+        ),
         Rule(Rule),
     }
 
+    #[derive(Debug, Clone)]
+    pub struct RustSnippet {
+        #[rust_sitter::leaf(text = "`")]
+        _start: (),
+        #[rust_sitter::leaf(pattern = r#"[^`]*"#, transform = |s| s.to_string())]
+        pub code: String,
+        #[rust_sitter::leaf(text = "`")]
+        _end: (),
+    }
```

Comment:
(Theoretically could break if you put some ```"as`df"``` in your code? Definitely fine for now though)

Reply:
yeah... eventually will want some escaping logic but punting to later